### PR TITLE
add contribution guidelines

### PR DESCRIPTION
**Pull request by christiemolloy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,194 @@
...
+
+Once your proposal has been approved by our team you can start developing. We suggest following this order:
+
+1. Write the title, description and sub-steps for each step in a document. This will help to organize your thoughts.
+2. It is not possible to run your code locally when developing in Katacoda so we suggest using CodePen or CodeSandbox to test the working code for your example.
+3. Once you feel confident in your steps and code examples, fork our repository: https://github.com/patternfly/training-scenarios.
+4. Add the new module to the repository.
+5. Push up the addition and submit a Pull Request.
+6. Tag at least two members of the PatternFly team to review your Pull Request.
+To see your changes live on Katacoda before being merged:
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
To see your changes on Katacoda before they're merged:
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,194 @@
...
+7. Submit a second Pull Request that merges changes from the new branch on your fork, to the master branch on your fork, and merge the changes.
```
Comment (abigaeljamie) [inline README.md]: These should be new steps, starting from 1. The previous set of steps (1 – 6) were dev instructions. Then we say, "To see your changes live," which involve a new set of instructions.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,194 @@
...
+8. Create a (Katacoda profile)[https://katacoda.com/profile/settings ] if you don’t have one.
+    1. Update your Git Scenario Repository to be your forked training-scenarios repository. E.g. https://github.com/[username]/training-scenarios
```
Comment (abigaeljamie) [inline README.md]: Let's not do a sub-numbers under numbers. They look confusing.

Comment (abigaeljamie) [inline README.md]:
```suggestion
    1. Update your Git Scenario Repository to be your forked training-scenarios repository (e.g., https://github.com/[username]/training-scenarios).
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,194 @@
...
+9. Copy the “Git Webhook Secret”
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
9. Copy the **Git Webhook Secret.**
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,194 @@
...
+10. In your forked training-scenarios repository add a new Github Webhook. It is under Settings > Webhooks.
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
10. In your forked training-scenarios repository, navigate to **Settings** > **Webhooks**, and add a new Github Webhook.
``` 
(We want to use the "Location, Action" structure when giving instructions. So we want to tell a user where to go (Settings > Webhooks) before telling them what to do (GitHub Webhook).

File: README.md [hunk 6]
```diff
@@ -1,3 +1,194 @@
...
+    1. Update the Payload URL to be https://editor.katacoda.com/scenarios/updated
+    2. Add your copied secret
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
    2. Add your copied secret.
```

File: README.md [hunk 7]
```diff
@@ -1,3 +1,194 @@
...
+4. Add the new module to the repository.
+5. Push up the addition and submit a Pull Request.
+6. Tag at least two members of the PatternFly team to review your Pull Request.
+To see your changes live on Katacoda before being merged:
+7. Submit a second Pull Request that merges changes from the new branch on your fork, to the master branch on your fork, and merge the changes.
+8. Create a (Katacoda profile)[https://katacoda.com/profile/settings ] if you don’t have one.
+    1. Update your Git Scenario Repository to be your forked training-scenarios repository. E.g. https://github.com/[username]/training-scenarios
+9. Copy the “Git Webhook Secret”
+10. In your forked training-scenarios repository add a new Github Webhook. It is under Settings > Webhooks.
+    1. Update the Payload URL to be https://editor.katacoda.com/scenarios/updated
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
    1. Update the Payload URL to **https://editor.katacoda.com/scenarios/updated**.
```

File: README.md [hunk 8]
```diff
@@ -1,3 +1,194 @@
...
+    2. Add your copied secret
+    3. Click: “Update webhook”
```
Comment (abigaeljamie) [inline README.md]:
```suggestion
    3. Click **Update webhook**.
```

File: README.md [hunk 9]
```diff
@@ -1,3 +1,194 @@
...
+11. Find more information (here)[https://katacoda.com/teach/git-hosted-scenarios].
```
Comment (abigaeljamie) [inline README.md]: This needs to be a descriptive, contextual hyperlink for accessibility purposes. We should avoid verbiage like "click here," as that's not good for accessibility.

File: README.md [hunk 10]
```diff
@@ -1,3 +1,194 @@
...
+
+## File structure
+
+### Update the category pathway
+
+Update the relevant `pathway.json` file so that the new module will be added to its category. Examples include: `html-css-pathway.json`, `react-components-pathway.json`, and `react-pathway.json`. You will need to add an ID, title, and description.
+
+### Add an index.json file for the module
+
+The `index.json` file is where the information about the module is defined and the structure is established. Below is a template `index.json` file that you should follow. (Katacoda provides more information)[https://www.katacoda.com/docs/scenarios/index-json].
```
Comment (gheake2) [inline README.md]:
```suggestion
The `index.json` file is where the information about the module is defined and the structure is established. Below is a template `index.json` file that should be followed. (Katacoda provides more information)[https://www.katacoda.com/docs/scenarios/index-json].
```

File: README.md [hunk 11]
```diff
@@ -1,3 +1,194 @@
...
+  "backend": {
+    "imageid": "node",
+    "port": 3000
+  }
+}
+```
+
+## Content and steps
+
+When writing out the steps for your tutorial please follow these guidelines:
```
Comment (gheake2) [inline README.md]:
```suggestion
When writing out the steps for the tutorial please follow these guidelines:
```

File: README.md (hunk 9 above)
Comment (gheake2) [inline README.md]:
```suggestion
11. Find more information (at Katacoda.com)[https://katacoda.com/teach/git-hosted-scenarios].
```

File: README.md (hunk 5 above)
Comment (gheake2) [reply README.md]: 11. In your forked training-scenarios repository add a new Github Webhook. It is under Settings > Webhooks.

File: README.md [hunk 12]
```diff
@@ -1,3 +1,194 @@
...
+
+### Estimated time
+
+* Purpose: To estimate the time that it will take the user to complete the module.
+
+### Introduction page
+
+* Purpose: To introduce the user to the module content. The introduction is always a critical part of any course. It is important to be brief, but clear about what information will be covered. The approximate length should be 3-5 sentences.
+    * Incorrect: PatternFly React charts are fun to use. (Not enough information about the module's content).
+    * Correct: PatternFly consists of isolated and modular structures that fall into three categories: Components: Components are modular and independent structures concerned with presentation. Layouts: Layouts allow for organizing and grouping their immediate children on the page. Demos: Demos illustrate how to assemble complex structures with components and layouts only. Utilities: Utilities are a set of classes that enable you to further customize and modify elements in your project without having to write any custom CSS.
```
Comment (gheake2) [inline README.md]:
```suggestion
    * Correct: PatternFly consists of isolated and modular structures that fall into three categories: Components: Components are modular and independent structures concerned with presentation. 
  
- Layouts: Layouts allow for organizing and grouping their immediate children on the page. 
- Demos: Demos illustrate how to assemble complex structures with components and layouts only. 
-  Utilities: Utilities are a set of classes that enable that customize and modify elements in the module without having to write any custom CSS.

```

File: README.md [hunk 13]
```diff
@@ -1,3 +1,194 @@
...
+
+### Step title
+
+* Purpose: To inform the user what step they are on within the module. Use sentence case, be clear and concise, be actionable, and use present tense.
+    * Incorrect: 1
```
Comment (gheake2) [inline README.md]:
```suggestion
    * Incorrect: Step 1
```

File: README.md [hunk 14]
```diff
@@ -1,3 +1,194 @@
...
+
+### Hints
+
+* Purpose: To provide the user with a clue about what they should do for that instruction. 
+
+### Feedback link
+
+* Purpose: To gather feedback from users after completing a module.
+    1. In the `finish.md` file, add a link to our module survey so that feedback can be recorded.
+    2. Use this link: https://redhatdg.co1.qualtrics.com/jfe/form/SV_bIRZRHYJyGsKBSt?Module=
```
Comment (gheake2) [inline README.md]:
```suggestion
    2. Use this [link to the survey](https://redhatdg.co1.qualtrics.com/jfe/form/SV_bIRZRHYJyGsKBSt?Module=)
```

File: README.md (hunk 12 above)
Comment (christiemolloy) [reply README.md]: we mentioned to use sentences here so why would we break them into bullets?

**Pull request merged**